launchable/commands/record/commit.py

```diff
@@ -53,16 +53,30 @@ def commit(ctx, source: str, executable: bool, max_days: int, scrub_pii: bool, i
         _import_git_log(import_git_log_output, ctx.obj.dry_run)
         return
 
+    cwd = os.path.abspath(source)
     try:
-        exec_jar(os.path.abspath(source), max_days, ctx.obj.dry_run)
+        is_shallow = subprocess.check_output(
+            ['git', 'rev-parse', '--is-shallow-repository'],
+            stderr=subprocess.DEVNULL,
+            cwd=cwd,
+            universal_newlines=True).strip()
+        if is_shallow == "true":
+            click.echo(click.style(
+                "Can't collect all commits from {} since it is the shallow repository. "
+                "Please use full clone or disable commit collection by --no-commit-collection option."
+                .format(cwd),
+                fg='yellow'),
```

Comment:
We'll exit as code=1 so, should use error color of `fg='red'` than warning color of `fg='yellow'`

Reply:
[Update message color](https://github.com/launchableinc/cli/pull/574/commits/b90aed90e6d82f6f0e38f7b17d4e229c21333f65)